samples/bluetooth/peripheral_uart/boards/pm_static_thingy53_nrf5340_cpuapp.yml

```
@@ -0,0 +1,41 @@
+app:
```

Could you change partitions configuration to reserve 64k for settings?
```suggestion
app:
  address: 0x10200
  region: flash_primary
  size: 0xdfe00
mcuboot:
  address: 0x0
  region: flash_primary
  size: 0x10000
mcuboot_pad:
  address: 0x10000
  region: flash_primary
  size: 0x200
mcuboot_primary:
  address: 0x10000
  orig_span: &id001
  - mcuboot_pad
  - app
  region: flash_primary
  size: 0xe0000
  span: *id001
mcuboot_primary_app:
  address: 0x10200
  orig_span: &id002
  - app
  region: flash_primary
  size: 0xdfe00
  span: *id002
settings_storage:
  address: 0xf0000
  region: flash_primary
  size: 0x10000
mcuboot_secondary:
  address: 0x00000
  size: 0xe0000
  device: MX25R64
  region: external_flash
external_flash:
  address: 0xe0000
  size: 0x720000
  device: MX25R64
  region: external_flash
```